backend/src/test/java/neuefische/capstone/bpmndiagram/BpmnDiagramIntegrationTest.java

```diff
@@ -0,0 +1,361 @@
+                                }
+                                """))
+                .andExpect(status().isCreated())
+                .andReturn().getResponse().getContentAsString();
+
+        String responseBody2 = mockMvc.perform(MockMvcRequestBuilders.post("/api/bpmndiagrams")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("""
+                                {
+                                    "name": "Create New Diagram",
+                                    "businessKey": "create-diagram",
+                                    "filename": "create-new-diagram.bpmn",
+                                    "version": 2,
+                                    "calledProcesses": null,
+                                    "commentText": null,
+                                    "commentTime": null,
+                                    "commentAuthor": null,
+                                    "customDiagram": true
+                                }
+                                """))
+                .andExpect(status().isCreated())
+                .andReturn().getResponse().getContentAsString();
+
+        BpmnDiagram responseObject1 = objectMapper.readValue(responseBody1, BpmnDiagram.class);
+        BpmnDiagram responseObject2 = objectMapper.readValue(responseBody2, BpmnDiagram.class);
+
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/bpmndiagrams/history/" + responseObject1.businessKey()))
+                .andExpect(status().isOk())
+                .andExpect(content().json("""
+                        [
+                        {
+                            "id": "<id1>",
+                            "name": "Create_Diagram",
+                            "businessKey": "create-diagram",
+                            "filename": "create-diagram.bpmn",
+                            "version": 1,
+                            "calledProcesses": null,
+                            "commentText": null,
+                            "commentTime": null,
+                            "commentAuthor": null,
+                            "customDiagram": true
+                        },
+                        {
+                            "id": "<id2>",
+                            "name": "Create New Diagram",
+                            "businessKey": "create-diagram",
+                            "filename": "create-new-diagram.bpmn",
+                            "version": 2,
+                            "calledProcesses": null,
+                            "commentText": null,
+                            "commentTime": null,
+                            "commentAuthor": null,
+                            "customDiagram": true
+                        }]
+                        """.replace("<id1>", responseObject1.id()).replace("<id2>", responseObject2.id())));
+    }
+
+    @Test
+    @DirtiesContext
+    void POSTBpmnDiagram_expect201() throws Exception {
```

Comment:
Lieber den anderen Test umbenennen zu "Post and Get", statt hier nochmal das gleiche zu testen